nbs/01_Tutorials/01_tutorial.ipynb

```diff
@@ -16,7 +16,7 @@
    "cell_type": "markdown",
```

Comment:
<p>You might want to mention a few words about if GitHub is necessary for nbdev, or how it’s related to nbdev currently.  This is a a very common question we get.  Perhaps you decide it’s too confusing to add it here, but just wanted to mention that incase you can think of a nice way to mention it here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/921/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='105'/>

Reply:
Good point. I'm going to merge without this for now since this tutorial is already so tightly coupled to GH, but this is something we should document somewhere